components/eamxx/src/physics/rrtmgp/scream_rrtmgp_interface.cpp

```diff
@@ -728,6 +766,8 @@ namespace scream {
             bool top_at_1 = p_lay_host(1, 1) < p_lay_host(1, nlay);
 
             k_dist.gas_optics(nday, nlay, top_at_1, p_lay_day, p_lev_day, t_lay_limited, gas_concs_day, optics, toa_flux);
+            k_dist.gas_optics(nday, nlay, top_at_1, p_lay_day, p_lev_day, t_lay_limited, gas_concs_day, optics_no_aerosols, toa_flux);
```

Comment:
This is a fairly expensive operation, and will be identical to the operation with `optics` with aerosol. I think it might be preferable to simple do a deep copy of the `optics` computed, unless this is meant to allow for changing gas optics as well (but we'd probably want to implement that a little differently to mimic the EAM-style diag calls).